### ISSUE TITLE
Can't see pages in the common directory

### ISSUE THREAD

**Issue by sbrl** [tldr-pages/tldr-node-client#95]
## Expected behaviour
Running `tldr inkscape` should yield a tldr page.

## Actual behaviour
```
Try updating with "tldr --update", or submit a pull request to:
https://github.com/tldr-pages/tldr
```

## Environment
| Key | Value |
|-------|---------|
| Operating system | `Linux Snowflake 4.2.0-16-generic #19-Ubuntu SMP Thu Oct 8 15:35:06 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux` |
| NodeJS version | `v5.9.0` |

**Comment by igorshubovych** [comment]
Have you tried `tldr --update`?

Because I see `inkscape` page.

**Comment by sbrl** [comment]
@igorshubovych Yep, I've tried that. Asciicast: https://asciinema.org/a/40517

**Comment by igorshubovych** [comment]
Try 2 commands and send output:

```bash
tldr --list -1 | grep inkscape
tldr --list-all -1 | grep inkscape
```

**Comment by sbrl** [comment]
```
sbrl@Snowflake:~$ tldr --list -1 | grep inkscape

  error: unknown option `-1'

sbrl@Snowflake:~$ tldr --list-all -1 | grep inkscape

  error: unknown option `--list-all'
```

**Comment by rubenvereecken** [comment]
Try updating `tldr` first:

```
npm update -g tldr
```

**Comment by sbrl** [comment]
Nope. Doesn't work.

```
sbrl@Snowflake:~$ sudo npm update -g tldr
[sudo] password for sbrl: 
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
/usr/local/bin/tldr -> /usr/local/lib/node_modules/tldr/bin/tldr
/usr/local/lib
└── tldr@1.3.0  (git+https://github.com/glortho/tldr-node-client.git#5e0678c60d8231c6ffdcb61729cc356106157961)

sbrl@Snowflake:~$ sudo npm update -g
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
/usr/local/bin/tldr -> /usr/local/lib/node_modules/tldr/bin/tldr
/usr/local/lib
└── tldr@1.3.0  (git+https://github.com/glortho/tldr-node-client.git#5e0678c60d8231c6ffdcb61729cc356106157961)

sbrl@Snowflake:~$ tldr --update
Updating...
sbrl@Snowflake:~$ tldr inkscape
Page not found
Try updating with "tldr --update", or submit a pull request to:
https://github.com/tldr-pages/tldr
sbrl@Snowflake:~$ tldr --list-all -1

  error: unknown option `--list-all'

sbrl@Snowflake:~$ tldr --list -1

  error: unknown option `-1'
```

**Comment by igorshubovych** [comment]
Ah, I see.
Just reinstall tldr

`sudo npm install -g tldr`

**Comment by sbrl** [comment]
Ouch, it didn't like that at _all_!

```
sbrl@Snowflake:~$ sudo npm install -g tldr
[sudo] password for sbrl: 
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
/usr/local/bin/tldr -> /usr/local/lib/node_modules/tldr/bin/tldr

> tldr@1.6.0 postinstall /usr/local/lib/node_modules/tldr
> node ./bin/tldr --update

Updating...
fs.js:584
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: EACCES: permission denied, open '/home/sbrl/.tldr/cache/LICENSE.md'
    at Error (native)
    at Object.fs.openSync (fs.js:584:18)
    at Object.fs.writeFileSync (fs.js:1224:33)
    at fCopyFile (/usr/local/lib/node_modules/tldr/node_modules/wrench/lib/wrench.js:275:16)
    at Object.exports.copyDirSyncRecursive (/usr/local/lib/node_modules/tldr/node_modules/wrench/lib/wrench.js:304:13)
    at /usr/local/lib/node_modules/tldr/lib/cache.js:56:12
    at Extract.<anonymous> (/usr/local/lib/node_modules/tldr/lib/remote.js:28:5)
    at emitNone (events.js:85:20)
    at Extract.emit (events.js:179:7)
    at DirWriter.<anonymous> (/usr/local/lib/node_modules/tldr/node_modules/unzip2/lib/extract.js:42:10)
- rimraf@2.4.5 node_modules/tldr/node_modules/cpr/node_modules/rimraf
- cpr@1.0.0 node_modules/tldr/node_modules/cpr
- deep-extend@0.4.1 node_modules/tldr/node_modules/deep-extend
- graceful-fs@3.0.8 node_modules/tldr/node_modules/fstream/node_modules/graceful-fs
- lodash@3.10.1 node_modules/tldr/node_modules/lodash
- glob@7.0.3 node_modules/tldr/node_modules/rimraf/node_modules/glob
- string@3.3.1 node_modules/tldr/node_modules/string
npm ERR! Linux 4.2.0-16-generic
npm ERR! argv "/usr/local/lib/node-v5.9.0-linux-x64/bin/node" "/usr/local/bin/npm" "install" "-g" "tldr"
npm ERR! node v5.9.0
npm ERR! npm  v3.7.3
npm ERR! code ELIFECYCLE

npm ERR! tldr@1.6.0 postinstall: `node ./bin/tldr --update`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the tldr@1.6.0 postinstall script 'node ./bin/tldr --update'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the tldr package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./bin/tldr --update
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs tldr
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls tldr
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/sbrl/npm-debug.log
```

Link to `npm-debug.log`: https://transfer.sh/ad2np/npm-debug.log

**Comment by igorshubovych** [comment]
See our FAQ on installation

**Comment by sbrl** [comment]
@igorshubovych Yep, that fixed it. Thanks for your help!
